test/e2e/storage/external/external.go

```diff
@@ -251,7 +251,7 @@ func (d *driverDefinition) SkipUnsupportedTest(pattern testpatterns.TestPattern)
 	switch pattern.SnapshotType {
 	case "":
 		supported = true
-	case testpatterns.DynamicCreatedSnapshot:
+	case testpatterns.DynamicCreatedSnapshot, testpatterns.PreprovisionedCreatedSnapshot:
```

Comment:
Hm I remember this change caused non-snapshot tests to be skipped for drivers that didn't enable the snapshot capability because the snapshot capability was added to the default test pattern. Do we still have that issue? cc @Jiawei0227 @chrishenzie

Reply:
Hmm, I dont remember if a fix for that is being made somewhere... 
But https://github.com/kubernetes/kubernetes/blob/0083786ca94a86bb455cb1c8eab7211739ca2ce6/test/e2e/storage/framework/testpattern.go#L114 on the master branch DefaultFsDynmicPV still has SnapshotType set.

Reply:
So in our current SkipInvalidDriverPatternCombination function, we did not check snapshot at all
https://github.com/kubernetes/kubernetes/blob/0083786ca94a86bb455cb1c8eab7211739ca2ce6/test/e2e/storage/framework/testsuite.go#L94

I remember the decision is that we will let the testsuite to check if it supports the pattern + driver. And all snapshot tests check the snapshot capability before testing. So it should be good in the master.

Reply:
https://github.com/kubernetes/kubernetes/pull/96843

Reply:
https://github.com/kubernetes/kubernetes/pull/96843/files#r533000567 :)

Reply:
So i should cherrypick 96843 this too?

Reply:
@msau42 for cla issue, do you know who we should contact for help? 

Also should I cherrypick  #96843 with this change too?

Reply:
Yes I think #96843 is needed to fix the tests for drivers that don't support snapshots.